doc/source/examples/unresolved-cfd-dem/dense-pneumatic-conveying/dense-pneumatic-conveying.rst

```diff
@@ -0,0 +1,634 @@
+
+We choose the `quadrature centred method (QCM) <../../../theory/multiphase/cfd_dem/unresolved_cfd-dem.html#the-quadrature-centered-method>`_  to calculate the void fraction. The ``l2 smoothing factor`` we choose is the square of twice the diameter of the particles.
+
+.. code-block:: text
+
+    subsection void fraction
+      set mode                = qcm
+      set read dem            = true
+      set dem file name       = dem
+      set l2 smoothing factor = 0.0001
+    end
+
+CFD-DEM
+~~~~~~~~~~
+
+The chosen drag model is Di Felice, and we use the Saffman lift force, the buoyancy force, and the pressure force. The coupling frequency is set to 100, which means that the DEM time step is 5e-6 s, for a Rayleigh critical time step of about 3.5%. The grad-div stabilization is used with a length scale of 0.084, the diameter of the pipe.
+
+.. code-block:: text
+
+   subsection cfd-dem
+     set grad div               = true
+     set drag model             = difelice
+     set saffman lift force     = true
+     set buoyancy force         = true
+     set pressure force         = true
+     set coupling frequency     = 100
+     set implicit stabilization = false
+     set grad-div length scale  = 0.084
+     set particle statistics    = true
+   end
+
+Non-linear Solver
+~~~~~~~~~~~~~~~~~
+
+We use the inexact Newton non-linear solver to minimize the number of time the matrix of the system is assembled. This is used to increase the speed of the simulation, since the matrix assembly requires significant computations.
+
+.. code-block:: text
+
+   subsection non-linear solver
+     subsection fluid dynamics
+       set solver           = inexact_newton
+       set matrix tolerance = 0.1
+       set reuse matrix     = true
+       set tolerance        = 1e-4
+       set max iterations   = 10
+       set verbosity        = quiet
+     end
+   end
+
+Linear Solver
+~~~~~~~~~~~~~
+
+.. code-block:: text
+
+   subsection linear solver
+     subsection fluid dynamics
+       set method                                = gmres
+       set max iters                             = 200
+       set relative residual                     = 1e-4
+       set minimum residual                      = 1e-6
+       set ilu preconditioner fill               = 0
+       set ilu preconditioner absolute tolerance = 1e-8
+       set ilu preconditioner relative tolerance = 1.00
+       set verbosity                             = quiet
+       set max krylov vectors                    = 200
+     end
+   end
+
+
+--------
+Results
+--------
+
+The results presented here are obtained from a custom post-processing code that is currently not provided with the example.
+
+Mass Flow Rate and Velocities
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Here we show the average velocities for the fluid, the slug and the particles in slug. The beta force, the averaged solid mass flow rate and the slug length over time are also shown. The shaded area represents the transient state.
```

Comment:
It is very approximate in this case since the simulation doesn't last long, this is when velocities fluctuate around the same value. I could have taken a more conservative start time (3.0 s) I agree. I could add a comment about that.

Reply:
It was a good question, I did change it, 4.0s is better even tho we only have 1 s of data to process. This is fine for the example :)